build/build.sln

```diff
@@ -743,6 +743,9 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "CoreGraphics", "CoreGraphics", "{0A79AFE5-2685-433C-BC78-A4AD09CD7BF8}"
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "CoreGraphics.UnitTests", "Tests\UnitTests\CoreGraphics\CoreGraphics.UnitTests.vcxproj", "{DA6C01EA-A22E-4807-BACE-63C5C6ABAE77}"
+	ProjectSection(ProjectDependencies) = postProject
```

Comment:
Interesting; I didn't think other dependencies added this block.

Can you remove it and make sure everything else works? We're trying to avoid expressing dependencies in SLN